### ISSUE TITLE
TG2-ISSUE_VERBATIMLATITUDE_NOTEMPTY

### ISSUE THREAD

**Issue by Tasilee** [tdwg/bdq#251]
| TestField | Value |
| ----- | ----- |
| **GUID** | 286fa10a-dab0-4f73-8732-76e379c932df |
| **Label** | ISSUE_VERBATIMLATITUDE_NOTEMPTY |
| **Description** | Is there a value in dwc:verbatimLatitude? |
| **TestType** | Validation |
| **Darwin Core Class** | Location |
| **Information Elements ActedUpon** | dwc:verbatimLatitude |
| **Information Elements Consulted** |  |
| **Expected Response** | POTENTIAL_ISSUE if dwc:verbatimLatitude is bdq:NotEmpty; otherwise NOT_ISSUE |
| **Data Quality Dimension** | Completeness |
| **Term-Actions** | VERBATIMLATITUDE_NOTEMPTY |
| **Parameter(s)** | |
| **Source Authority** |  |
| **Specification Last Updated** | 2024-04-02 |
| **Examples** |  [dwc:verbatimLatitude="42 45 04.03S": Response.status=RUN_HAS_RESULT, Response.result=POTENTIAL_ISSUE, Response.comment="dwc:verbatimLatitude is bdq:NotEmpty"] |
| | [dwc:verbatimLatitude="": Response.status=RUN_HAS_RESULT, Response.result=NOT_ISSUE, Response.comment="dwc:verbatimLatitude is bdq:Empty"] |
| **Source** | TG2 |
| **References** |  |
| **Example Implementations (Mechanisms)** |  |
| **Link to Specification Source Code** |  |
| **Notes** | This bdq:Supplementary test is not regarded as CORE (cf. bdq:CORE) because of one or more of the reasons: not being widely applicable; not informative or likely to return a high percentage of either bdq:COMPLIANT or bdq:NOT_COMPLIANT results (cf bdq:Response.result). A Supplementary test may be implemented as CORE when a suitable use case exists. |

**Comment by chicoreus** [comment]
See comment in #252 likely do not implement.   Many data are correctly expected to have no data in dwc:verbatimLatitude and dwc:verbatimLongitude.

**Comment by chicoreus** [comment]
We probably want to collect the verbatim notempty tests into a single UseCase, not simply put them out as supplementary. Substantial more thought is needed on what the data quality needs are that would make us data is unfit for use when verbatim data are not present.

**Comment by Tasilee** [comment]
I agree that a NOT_COMPLIANT response on many of the verbatim terms has little value: An issue we missed on our 'positive focus'. But under the current definition, doesn't this fit "Supplementary" more than DO NOT IMPLEMENT?

**GENERAL COMMENTS**: All of us have agreed that adding the additional found tests as Supplementary or DNI (or something else not CORE) is a side issue, but... @chicoreus said that we have learnt a few new lessons through the process, one being how we consider verbatim term tests. So over and above documenting our work, our decisions and flagging warnings to future implementers, it has been useful. 

But, we need to get back to core work.  We cannot progress until we have agreed on the status of these new 'tests'. We first need to agree on definitions and use those to classify them into one of the categories CORE, Supplementary, DO NOT IMPLEMENT or 'ignore' or whatever.

If, through a quick edit, we can adapt these tests to be more informative, fine. Go for it.

FYI: Below is part of a table I found on Google docs (dated July 2020). About 95% of GBIF records don't have a value in the verbatim terms.

term | number | %
-- | -- | --
verbatimCoordinates | 62,500,566 | 4.4
verbatimCoordinateSystem | 48,277,584 | 3.4
verbatimDepth | 4,518,745 | 0.3
verbatimElevation | 10,949,747 | 0.8
verbatimEventDate | 78,608,182 | 5.6
verbatimLatitude | 57,343,684 | 4.1
verbatimLocality | 63,125,555 | 4.5
verbatimLongitude | 57,327,356 | 4.1
verbatimSRS | 21,082,516 | 1.5

**Comment by chicoreus** [comment]
@Tasilee I think our categories are: 

- Mature Tests included in the standard
-- Tests for the CORE use case
-- Tests for second taxon oriented use case 
- Probably mature tests listed as supplemental
- Immature tests that those doing further work should think out carefully
- Do not Implement tests where we have found problems that preclude implementation

**Comment by Tasilee** [comment]
Thanks @chicoreus. I like the rendering but I think we need to refine those categories by concise definitions (clear and non-overlapping), so that we are confident on how to evaluate these additional tests.

**Comment by Tasilee** [comment]
This test is similar to #247, #248, #249, #252 and as I commented on #247, all these should be Supplementary (and Closed) for now as they are "not informative" as they stand, and I would not like us to be further distracted (q.v. @chicoreus comment on similar tests "...requires substantial more thought is needed..."). There could well be use cases, and we have commented adequately to inform future use.
